easybuild/easyconfigs/a/aNCI/aNCI-2.0-iccifort-2019.5.281.eb

```
@@ -8,23 +8,32 @@ description = """Non-covalent interaction (NCI) for MD trajectories"""
 
 toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
 
-source_urls = ['https://www.lct.jussieu.fr/pagesperso/contrera/']
-sources = [{'filename': SOURCE_ZIP, 'download_filename': 'anci-RC.zip'}]
-checksums = ['d57c2969fc180c687c946a0a1ee393fe1ffc252b3534337bc71e4d95acbbf6dd']
+source_urls = [
+    'https://www.lct.jussieu.fr/pagesperso/contrera/',
+    'https://github.com/juliacontrerasgarcia/nciplot/archive/',
+]
+sources = [
+    {'filename': SOURCE_ZIP, 'download_filename': 'anci-RC.zip'},
+    {'filename': '834af2e.tar.gz', 'extract_cmd': 'tar -xzf %s -C %(namelower)s'},
```

Two suggestions here
1. use a variable for the commit of `NCIPLOT` and add its name to the source file
2. only extract the `dat` folder from the tar file
```suggestion
    {'filename': '%s.tar.gz' % local_nciplot_commit, 'download_filename':  'NCIPLOT-%s.tar.gz' % local_nciplot_commit, 
     'extract_cmd': 'tar -xzf %s -C %(namelower)s --strip-components=1 */dat'},
```